queries/highlights.scm

```diff
@@ -9,12 +9,10 @@
   (null)
   (na)
 ] @constant.builtin
-(string) @string
 
-(escape_sequence) @constant.character.escape
+((string) @string (#match? @string "^\".+?\""))
 
+(escape_sequence) @constant.character.escape
 
 ; Punctuation
 (delimiter) @punctuation.delimiter
-(quote) @punctuation.bracket
```

Comment:
I want to keep brackets distinct from the string content. Please keep this line.